### PR TITLE
fix(events): wait for all intermediate statuses in PollCommandExecution

### PR DESCRIPTION
**Pull request by jisung-02**
## Summary

`exec` and non-interactive `websh` (e.g. `alpacon websh server ls`) returned empty output because `PollCommandExecution` only waited for `running` and `acked` statuses. Commands go through several intermediate states before producing a result — and the CLI was returning immediately with an empty result on the first non-matching status.

## Changes

- Add `queued`, `scheduled`, `delivered`, and `verifying` to the keep-polling status set in `PollCommandExecution`
- Add table-driven test cases covering full state transitions: `queued → delivered → running → success`, `scheduled → queued → success`, `verifying → running → success`
- Fix `for i := 0; i < 25; i++` → `for i := range 25` (rangeint lint)

## Root cause

Commit `2be242f` adapted the polling logic from nested `{"text": "Acked"}` to flat `"acked"` string format but only carried over `acked` as the keep-polling case, missing all other intermediate states. With the recent addition of command verification (AI signing), commands now pass through `verifying` before execution, making the issue consistently reproducible.

### Universal
- [x] Code follows project style guidelines
- [x] Self-reviewed the code
- [x] No hardcoded secrets or credentials

### Go
- [x] Error handling is complete
- [x] Tests include edge cases

### CODE REVIEW

File: api/event/event.go [hunk 1]
```diff
@@ -128,7 +128,7 @@ func PollCommandExecution(ac *client.AlpaconClient, cmdId string) (EventDetails,
 			}
 
 			switch response.Status {
-			case "running", "acked":
+			case "queued", "scheduled", "delivered", "verifying", "running", "acked":
 				timer.Reset(5 * time.Minute)
 				continue
```
[the remark this answers is not in the document]
Comment (jisung-02) [reply api/event/event.go:133]: This is not an issue — since Go 1.23, Timer.Reset guarantees that stale values are drained from the channel before it returns. This project uses Go 1.25.7.


https://go.dev/doc/go1.23#timer-changes

**Review by eunyoung14 — APPROVED** [review]
LGTM
